daemon/cmd/statedb.go

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+
+	. "github.com/cilium/cilium/api/v1/server/restapi/statedb"
+	"github.com/cilium/cilium/pkg/statedb"
+)
+
+type getStateDBDump struct {
+	db statedb.DB
+}
+
+func (h *getStateDBDump) Handle(params GetStatedbDumpParams) middleware.Responder {
+	return middleware.ResponderFunc(func(w http.ResponseWriter, _ runtime.Producer) {
+		h.db.WriteJSON(w)
```

Comment:
does swagger handle setting http response codes?

Reply:
Yep and the content-type too (similar usage you can see in `daemon/cmd/map.go`. This could also be written with the `GetStatedbDumpOK` which takes an `io.Reader`, but then I'd need a pipe and etc. This is much shorter and more obvious.